downstream/modules/devtools/proc-rhdh-configure-aap-details.adoc

```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="rhdh-configure-aap-details_{context}"]
+= Configuring Ansible Automation Platform details
+
+The Ansible plug-ins query your {PlatformNameShort} subscription status with the controller API using a token.
+
+[NOTE]
+====
+The Ansible plug-ins continue to function regardless of the {PlatformNameShort} subscription status.
+====
+
+.Procedure
+
+. Create a Personal Access Token (PAT) with “Read” scope in automation controller, following the
+link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_controller_user_guide/index#assembly-controller-applications[Applications] section of the _Automation controller user guide_. 
```

Comment:
This link doesn't work for me but I can't see whats (if anythings) wrong with it?